the22mastermind/questioner
# Ft user view meetup details 162733217 endpoint

#### What does this PR do?
Add view a single meetup functionality with database
#### Description of Task to be completed?
View details of a single meetup
#### How should this be manually tested?
1. After cloning repo, cd into it and run 'npm run start'
2. Using POSTMAN enter in URL: http://localhost:5000/api/v1/meetups/{meetupID} and you should see details of that meetup if it exists or a not found error.
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
#162733217 https://www.pivotaltracker.com/story/show/162733217
#### Screenshots (if appropriate)
#### Questions:
